static/app/views/issueList/overview.tsx

```diff
@@ -527,8 +543,20 @@ class IssueListOverview extends Component<Props, State> {
       ...this.getEndpointParams(),
       limit: MAX_ITEMS,
       shortIdLookup: 1,
+      savedSearch: this.props.organization.features.includes('issue-stream-performance')
+        ? this.props.savedSearchLoading
+          ? 0
+          : 1
+        : 1,
```

Comment:
maybe lets give these constants ENABLED or something. kinda hard to read without context